Griddly.Mvc/GriddlyExtensions.cs

```diff
@@ -530,12 +530,23 @@ private static GriddlyContext GetOrCreateGriddlyContext(RouteData routeData, Htt
 
                 sortFields = GriddlyResult.GetSortFields(items);
 
+#if NETFRAMEWORK
+                string cookieSuffix = (controller.ControllerContext.RouteData.Values["GridIdentifier"] as string);
+#else
+                string cookieSuffix = (routeData.Values["GridIdentifier"] as string);
+#endif
+
+                if (string.IsNullOrEmpty(cookieSuffix))
+                    cookieSuffix = items["GridIdentifier"];
+
+                cookieSuffix = !string.IsNullOrEmpty(cookieSuffix) ? "_" + cookieSuffix.ToLower() : string.Empty;
```

Comment:
This makes sense, but I think we should use a short querystring param name. Maybe `gid` or something like that.

Reply:
Just wanted to understand what is driving factor of reaming it to gid? I prefer long variable name so that it convey the usage without extra details (It could be just me) please advice.

Reply:
Usually I would too, but since this is part of the route data which is transmitted in the URL I prefer short.

Reply:
I have raised the new PR: https://github.com/programcsharp/griddly/pull/125 for all these changes